tests/e2e/cucumber/features/journeys/kindergarten.oc10.feature

```diff
@@ -94,6 +94,29 @@ Feature: Kindergarten can use web to organize a day
       | Pre-Schools Pirates | groups                               | folder |
       | Teddy Bear Daycare  | groups                               | folder |
       | groups              |                                      | folder |
+    And "Alice" deletes the following resources using the batch action
+      | resource            | from                                 |
+      | lorem.txt           | groups/Kindergarten Koalas/meal plan |
+      | lorem-big.txt       | groups/Kindergarten Koalas/meal plan |
+      | data.zip            | groups/Pre-Schools Pirates/meal plan |
+      | lorem.txt           | groups/Pre-Schools Pirates/meal plan |
+      | lorem-big.txt       | groups/Pre-Schools Pirates/meal plan |
+      | data.zip            | groups/Teddy Bear Daycare/meal plan  |
+      | lorem.txt           | groups/Teddy Bear Daycare/meal plan  |
+      | lorem-big.txt       | groups/Teddy Bear Daycare/meal plan  |
+    And "Alice" deletes the following resources using the sidebar panel
+      | resource            | from                                 |
+      | meal plan           | groups/Kindergarten Koalas           |
+      | meal plan           | groups/Pre-Schools Pirates           |
+      | meal plan           | groups/Teddy Bear Daycare            |
```

Comment:
I think it's better if we delete everything through a batch action. I doubt an actual user will switch between the sidebar and batch action while deleting everything.